src/hashtable.py

```diff
@@ -84,7 +100,8 @@ def resize(self):
 
         Fill this in.
         '''
-        pass
+        for i in range(self.capacity):
```

Comment:
This doesn't seem right! I know, the test are Passing but the function seems wrongs